### DISCUSSION TITLE
$filter operators and wildcards

### DISCUSSION THREAD

**Discussion by BumpaRoy** [Azure/data-api-builder#1730] [Q&A]
Is there a "contains" operator (like graphQL) equivalent in REST?  Is there a wildcard?

**Comment by seantleonard** [comment]
At the moment, no. Can you see if https://github.com/Azure/data-api-builder/issues/936 aligns with your ask? If not, feel free to add details to that issue.

**Reply by seantleonard** [reply]
For others seeing this thread, the current operators supported for REST are listed here: https://learn.microsoft.com/azure/data-api-builder/rest#filter
